.devcontainer/docker-compose-local.yaml

```diff
@@ -79,12 +79,11 @@ services:
       - POSTGRES_USER=admin
       - POSTGRES_PASSWORD=password
       - POSTGRES_DB=lumigator
-      - AWS_HOST=localstack
+      - S3_ENDPOINT_URL=http://localstack:4566
       - AWS_ACCESS_KEY_ID=test
       - AWS_SECRET_ACCESS_KEY=test
       - AWS_DEFAULT_REGION=us-east-2
       - AWS_ENDPOINT_URL=http://localstack:4566
```

Comment:
We have S3_ENDPOINT_URL and AWS_ENDPOINT_URL pointing to the same address, are these different variables?

Reply:
Yup, works for me and serves me right for not re-reading 🤦‍♀️

Reply:
to be fair it was a very long description :D I'll keep them smaller if possible! thanks!!

Reply:
No, no, this is good context and will be good to keep for reference, please keep writing these 🙏